tests/acceptance/new.spec.ts

```diff
@@ -166,4 +166,12 @@ describe('Acceptance: ng new', function () {
       .then(done, done.fail);
   });
 
+  it('should specify a version of the CLI', (done) => {
+    return ng(['new', 'FooApp', '--skip-install', '--skip-git']).then(() => {
+      const pkgJson = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+      expect(pkgJson.devDependencies['@angular/cli']).toMatch(/\d.\d.\d/);
```

Comment:
`.` in regexes are any characters. This would be better as the following: `/\d+\.\d+\.\d+/`, as there might be more digits.